lms/djangoapps/philu_overrides/branding/api.py

```diff
@@ -32,7 +32,7 @@ def is_enabled():
     return BrandingApiConfig.current().enabled
 
 
-def get_auth_footer(is_secure=True):
+def get_auth_footer(is_secure=True):  # pylint: disable=unused-argument
```

Comment:
Do we need to suppress this warning, or can we remove the argument `is_secure=True`?

The argument is not being used in the function, and the function is called only from `src/philu-edx-theme/edx-platform/philu/lms/templates/auth_footer.html`, line number 6:

`<% footer = get_auth_footer(is_secure=is_secure) %>`

We can remove the argument being passed from here as well, if it does not cause any problems.